package.json

```diff
@@ -33,9 +33,9 @@
   },
   "devDependencies": {
     "@angular-builders/custom-webpack": "^14.0.0",
-    "@angular-devkit/build-angular": "^14.1.0",
-    "@angular-devkit/core": "^14.1.0",
-    "@angular-devkit/schematics": "^14.1.0",
```

Comment:
Undo changes to this file.

Reply:
Then the build won't pass due to `@schematics/angular@4.2.0` updating their `typescript` dependency to a version this library doesn't support. I've only changed this because there's no lock file, but I can't undo it without a solution. You can see the build failure caused by the mismatch [here](https://dev.azure.com/ng-zorro/NG-ZORRO/_build/results?buildId=4936&view=logs&j=8680be89-378c-5901-27b8-79bf589da266&t=7cc10bac-49f7-5748-d4ee-64732cd3a79d).

Reply:
@simplejason I see the recent PRs run failed. This change can resolve this, what do you think of this?

Reply:
@UncleDave Could you open another PR for fixing CI? And we merge that first

Reply:
I also did what is in #7637, which I will remove from this PR in favour of that one.

Reply:
Dependency changes moved to #7640 
This PR should build after #7637 & #7640 are merged.